client/interfaces.go

```diff
@@ -314,6 +314,14 @@ type InstanceServer interface {
 	UpdateClusterMember(name string, member api.ClusterMemberPut, ETag string) (err error)
 	RenameClusterMember(name string, member api.ClusterMemberPost) (err error)
 
+	// Warning functions
+	GetWarnings() ([]api.Warning, error)
+	GetWarning(UUID string) (*api.Warning, error)
+	AcknowledgeAllWarnings() error
+	AcknowledgeWarning(UUID string) error
+	DeleteAllWarnings() error
```

Comment:
Drop all of those and replace with just a single:
`UpdateWarning(UUID string, api.WarningPut, ETag string) error`